config/sidebar-footer.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+      "source": "resources/faq.mdx",
+      "label": "FAQ",
+      "slug": "faq"
+    },
+    {
+      "to": "/",
+      "label": "Back to meilisearch.com",
+      "slug": "landing"
```

Comment:
Perhaps an ignorant question: I'm wondering what this `slug` property is actually doing, since there is no route being generated (to my knowledge).

Reply:
That's a good question, I answered that on Slack but forgot to mention it here. We would use the `slug` to match each item with its respective icon, and in the case it doesn’t have a `source` (like back to [meilisearch.com](http://meilisearch.com/)) the `to` field is used as the link for it. (It could be an absolute url as `https://blog.meilisearch.com/` too)